src/helpers/helpers.math.js

```diff
@@ -23,7 +23,7 @@ export const sign = Math.sign;
  */
 export function niceNum(range) {
   const niceRange = Math.pow(10, Math.floor(log10(range)));
-  const fraction = range / niceRange;
+  const fraction = range / niceRange - 0.000001;
```

Comment:
Not really liking this, but I guess it does the job.

Reply:
Yeah, it's not great especially because it's hard coded. I wasn't sure what a more correct value would be though